gui/src/loader.rs

```diff
@@ -225,6 +225,15 @@ impl Loader {
 
         if let Some(bitcoind) = &self.internal_bitcoind {
             bitcoind.stop();
+        } else if self.gui_config.start_internal_bitcoind {
```

Comment:
What if it gets called while we are performing the `connect`? This would be an infinite loop. I think we should set a flag right before calling `start_bitcoind_and_daemon`.